lib/meilisearch-rails.rb

```diff
@@ -682,8 +682,13 @@ def ms_index_uid(options = nil)
         options ||= meilisearch_options
         global_options ||= MeiliSearch::Rails.configuration
 
-        name = options[:index_uid] || model_name.to_s.gsub('::', '_')
-        name = "#{name}_#{::Rails.env}" if global_options[:per_environment]
+        name = options[:index_uid] || model_name.to_s.gsub('::', '_').downcase
```

Comment:
This implementation is not ready. It is also missing some new use cases of testing. 
I'll convert it to a draft. Ask for my review when you're ready, thanks :)